pages/about.tsx

```diff
@@ -36,15 +37,134 @@ export default function About() {
+            <Heading as="h4">Running in Development</Heading>
+            <Code language="bash">npm run dev</Code>
+            <Heading as="h4">Building in Production</Heading>
+            <Code language="bash">npm run build</Code>
+            <Heading as="h4">Configuration</Heading>
+            Canopy IIIF uses a default configuration{" "}
+            <code>config/.default/canopy.default.json</code> for demonstration
+            purposes if a custom one is not set. The build process will read
+            from a custom configuration file at <code>config/canopy.json</code>{" "}
+            if it exists.
+            <Heading as="h5">Create a Custom Configuration</Heading>
+            <ol>
+              <li>
+                Find your <code>config/</code> directory
+              </li>
+              <li>
+                Rename <code>canopy.sample.json</code> to{" "}
+                <code>canopy.json</code>
+              </li>
+              <li>
+                Updates should be made to both the <strong>prod</strong> and{" "}
+                <strong>dev</strong> configurations. These configurations can
+                match eachother; however in some cases, development speed can be
+                aided by targeting a <em>smaller</em> IIIF Collection{" "}
+                <code>id</code> as a fixture.
+              </li>
+            </ol>
+            <Heading as="h5">Example Configuration</Heading>
+            <Code language="json">{`{
+  "prod": {
+    "collection": "https://iiif.bodleian.ox.ac.uk/iiif/collection/hobhouse",
+    "featured": [
+      "https://iiif.bodleian.ox.ac.uk/iiif/manifest/8da97e8c-4e12-457d-aad8-3327b3aec183.json",
+    ],
+    "metadata": ["Title"]
+  },
+  "dev": {
+    "collection": "https://iiif.bodleian.ox.ac.uk/iiif/collection/hobhouse",
+    "featured": [
+      "https://iiif.bodleian.ox.ac.uk/iiif/manifest/8da97e8c-4e12-457d-aad8-3327b3aec183.json",
+    ],
+    "metadata": ["Title"]
+  }
+}`}</Code>
+            <Heading as="h5">Configuration Options</Heading>
+            <Heading as="h6">Label as Site Title</Heading>
+            <p>
+              The Canopy IIIF site title is the Collection label of the set{" "}
+              <code>collection</code> resource. You can override this by
+              providing a valid <a href="">Presentation 3.0 label property</a>.
+            </p>
+            <Code language="json">{`{
+  "prod": {
+    "label": { 
+      none: ["Hobhouse"]
+    },
+    "collection": "https://iiif.bodleian.ox.ac.uk/iiif/collection/hobhouse",
+    "featured": [
+      "https://iiif.bodleian.ox.ac.uk/iiif/manifest/8da97e8c-4e12-457d-aad8-3327b3aec183.json",
+    ],
+    "metadata": ["Title"]
+  },
+  ...
+}`}</Code>
+            <Heading as="h6">Curating Featured Manifests</Heading>
+            <p>
+              You can inform Canopy IIIF of featured Manifests by providing an
+              array of ids. These must be within the referenced{" "}
```

Comment:
We probably should be explicit here that these should be Images or have an image canvas at 0.